---
 src/77-mm-usb-device-blacklist.rules |    5 +++++
 1 file changed, 5 insertions(+)

Index: modemmanager-0.7.990/src/77-mm-usb-device-blacklist.rules
===================================================================
--- modemmanager-0.7.990.orig/src/77-mm-usb-device-blacklist.rules	2013-03-14 23:37:21.866210414 +0100
+++ modemmanager-0.7.990/src/77-mm-usb-device-blacklist.rules	2013-03-14 23:37:21.802210413 +0100
@@ -88,4 +88,9 @@
 # QinHeng Electronics HL-340
 ATTRS{idVendor}=="1a86", ATTRS{idProduct}=="7523", ENV{ID_MM_DEVICE_IGNORE}="1"
 
+# Arduino devices, FT-232-based series and Atmega8U2 (Uno)
+ATTRS{idVendor}=="0403", ATTRS{idProduct}=="6001", ENV{ID_MM_DEVICE_IGNORE}="1"
+ATTRS{idVendor}=="03eb", ATTRS{idProduct}=="204b", ENV{ID_MM_DEVICE_IGNORE}="1"
+ATTRS{idVendor}=="2341", ENV{ID_MM_DEVICE_IGNORE}="1"
+
 LABEL="mm_usb_device_blacklist_end"
