From: Alexander Sack <asac@ubuntu.com>
Date: Mon, 3 Feb 2014 09:50:59 +0100
Subject: arduino-blacklist

---
 src/77-mm-usb-device-blacklist.rules | 5 +++++
 1 file changed, 5 insertions(+)

Index: modemmanager/src/77-mm-usb-device-blacklist.rules
===================================================================
--- modemmanager.orig/src/77-mm-usb-device-blacklist.rules	2014-09-12 19:19:23.861269695 +0200
+++ modemmanager/src/77-mm-usb-device-blacklist.rules	2014-09-12 19:19:23.857269695 +0200
@@ -137,4 +137,9 @@
 # QinHeng Electronics HL-340
 ATTRS{idVendor}=="1a86", ATTRS{idProduct}=="7523", ENV{ID_MM_DEVICE_IGNORE}="1"
 
+# Arduino devices, FT-232-based series and Atmega8U2 (Uno)
+ATTRS{idVendor}=="0403", ATTRS{idProduct}=="6001", ENV{ID_MM_DEVICE_IGNORE}="1"
+ATTRS{idVendor}=="03eb", ATTRS{idProduct}=="204b", ENV{ID_MM_DEVICE_IGNORE}="1"
+ATTRS{idVendor}=="2341", ENV{ID_MM_DEVICE_IGNORE}="1"
+
 LABEL="mm_usb_device_blacklist_end"
