---
 src/77-mm-usb-device-blacklist.rules |    5 +++++
 1 file changed, 5 insertions(+)

Index: modemmanager/src/77-mm-usb-device-blacklist.rules
===================================================================
--- modemmanager.orig/src/77-mm-usb-device-blacklist.rules	2014-03-27 13:19:50.264031206 +0100
+++ modemmanager/src/77-mm-usb-device-blacklist.rules	2014-03-27 13:19:50.260031577 +0100
@@ -96,4 +96,9 @@
 # QinHeng Electronics HL-340
 ATTRS{idVendor}=="1a86", ATTRS{idProduct}=="7523", ENV{ID_MM_DEVICE_IGNORE}="1"
 
+# Arduino devices, FT-232-based series and Atmega8U2 (Uno)
+ATTRS{idVendor}=="0403", ATTRS{idProduct}=="6001", ENV{ID_MM_DEVICE_IGNORE}="1"
+ATTRS{idVendor}=="03eb", ATTRS{idProduct}=="204b", ENV{ID_MM_DEVICE_IGNORE}="1"
+ATTRS{idVendor}=="2341", ENV{ID_MM_DEVICE_IGNORE}="1"
+
 LABEL="mm_usb_device_blacklist_end"
