From: Alexander Sack <asac@ubuntu.com>
Date: Mon, 3 Feb 2014 09:50:59 +0100
Subject: linux-default-usb-id

---
 src/77-mm-usb-device-blacklist.rules | 3 +++
 1 file changed, 3 insertions(+)

Index: modemmanager/src/77-mm-usb-device-blacklist.rules
===================================================================
--- modemmanager.orig/src/77-mm-usb-device-blacklist.rules	2014-09-12 19:19:26.665269596 +0200
+++ modemmanager/src/77-mm-usb-device-blacklist.rules	2014-09-12 19:19:26.661269596 +0200
@@ -142,4 +142,7 @@
 ATTRS{idVendor}=="03eb", ATTRS{idProduct}=="204b", ENV{ID_MM_DEVICE_IGNORE}="1"
 ATTRS{idVendor}=="2341", ENV{ID_MM_DEVICE_IGNORE}="1"
 
+# Netchip Technology, Inc. Linux-USB Serial Gadget (CDC ACM mode)
+ATTRS{idVendor}=="0525", ATTRS{idProduct}=="a4a7", ENV{ID_MM_DEVICE_IGNORE}="1"
+
 LABEL="mm_usb_device_blacklist_end"
